common-ui/src/main/res/values/design-system-theming.xml

```diff
@@ -136,14 +136,13 @@
         <!--Design System Colors theme independent-->
         <item name="daxColorDim">@color/black60</item>
         <item name="daxColorWhite">@color/white</item>
+        <item name="daxColorBlack">@color/black84</item>
```

Comment:
I think this is quite confusing 🤔 Is this a pattern? Where should we use `daxColorBlack` (not black)

Reply:
What do you mean? `daxColorBlack` is defined in the Base Theme because it always has the same value. Light and Dark theme use the same value of `daxColorBlack`.

We need to start moving away from using `@color/black` (and any other @color/whatever)  in our resources files and start using the semantic name ?`attr/daxColorBlack`.